client/src/components/CarouselDots/CarouselDots.css

```diff
@@ -0,0 +1,28 @@
+.CarouselDots {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: flex-start;
+  transition: all 0.5s ease;
+}
+
+
+.CarouselDot {
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  margin: 10px;
+  text-indent: -999px;
+  cursor: pointer;
+  background-color: var(--white);
+  border-radius: 20px;
+}
+
+
+.CarouselDot__Border {
+  border: 2px solid var(--purple);
+}
+
+.CarouselDot__Active {
+  background-color: var(--purple)
+}
```

Comment:
Modifier/element names should be lower case (i.e. `CarouselDot__border`). Also, `active` is more of a modifier - so maybe `.CarouselDot--active`